openlibrary/catalog/edition_merge/templates/index.html

```diff
@@ -27,12 +27,12 @@ <h1>Merge duplicate editions</h1>
 <a href="?page={{ i + 1 }}">{{ i + 1}}</a>
 {% endif %}
 {% endfor %}
-<p>
+
 {% endmacro %}
 
 {{ pager() }}
 
-{{ total }} items need to be merged ({{ easy }} easy)<p>
+{{ total }} items need to be merged ({{ easy }} easy)
```

Comment:
should this entire line be inside a p? i'm wondering if the <p> at the end was meant to close the one inside the macro originally?

e.g. 
`<p> {{ total }} items need to be merged ({{ easy }} easy)</p>`